### DISCUSSION TITLE
Ciallo～(∠・ω< )⌒☆~

### DISCUSSION THREAD

**Discussion by xiao7hxh** [share121/inter-knot#2412] [其他]
<a href="http://yoshino.store">Ciallo～(∠・ω< )⌒☆~</a>

**Comment by cyrilcc-z** [comment]
Ciallo～(∠・ω< )⌒☆~

**Comment by God-Arch** [comment]
柚子厨蒸鹅心<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/7cb87b286c3d062300d31224508c0c9e_8768116406867668880.png" alt="丹恒 惊" width="75" >
